x-pack/plugins/observability/public/embeddable/slo/slo_configuration.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useMemo } from 'react';
+import {
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { SloSelector } from './slo_selector';
+
+import type { SloConfigurationProps, EmbeddableSloProps } from './types';
+
+export function SloConfiguration({ onCreate, onCancel }: SloConfigurationProps) {
+  const [selectedSlo, setSelectedSlo] = useState<EmbeddableSloProps>();
+  const updatedProps = useMemo(() => {
+    return {
+      ...selectedSlo,
+      title: i18n.translate('xpack.observability.sloEmbeddable.displayTitle', {
+        defaultMessage: 'SLO Overview',
+      }),
```

Comment:
Yep. This is the default title of the panel. Otherwise it shows `No Title`. Indeed it looks a bit odd in the slo configuration file.
<img width="364" alt="Screenshot 2023-09-22 at 21 38 07" src="https://github.com/elastic/kibana/assets/2852703/f06092c4-009e-43c8-834b-66c939a1ea05">

I found [this](https://github.com/elastic/kibana/tree/main/src/plugins/embeddable#input-state) so I can set the title directly in the `slo_embeddable` file

```  
setTitle(title: string) {
    this.updateInput({ title });
  }
```

```
this.setTitle(this.input.title || 18n.translate('xpack.observability.sloEmbeddable.displayTitle', {
        defaultMessage: 'SLO Overview',
      }));
```